### PR TITLE
Updating README and Adding CHANGELOG, CONTRIBUTING & LICENSE files

### PR DESCRIPTION
**Pull request by arcanedev-maroc**
See #531

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,16 @@
 
 In this repository, you can find the lang files for the framework PHP, [Laravel 4&5](http://www.laravel.com).
 
-## How add languages in my app ?
+## Install
 
-### Installation by GitHub
+#### Via Composer
+ * For Laravel 4 : add `"caouecs/laravel4-lang": "~1.0"` in your `composer.json` in "require" or run *composer require caouecs/laravel4-lang*
```
Comment (caouecs) [inline README.md]: you forget *For Laravel 5.2 : add `"caouecs/laravel-lang": "~3.0"` in your `composer.json` in "require"*

**Comment by arcanedev-maroc** [comment]
Done :+1:

File: README.md [hunk 2]
```diff
@@ -2,9 +2,17 @@
 
 In this repository, you can find the lang files for the framework PHP, [Laravel 4&5](http://www.laravel.com).
 
-## How add languages in my app ?
+## Install
 
-### Installation by GitHub
+#### Via Composer
+ * For Laravel 5.2 : add `"caouecs/laravel-lang": "~3.0"` in your `composer.json` in "require"
```
Comment (overtrue) [inline README.md]: maybe `caouecs/laravel-lang` -> `caouecs/laravel4-lang` ? 😄

**Comment by caouecs** [comment]
Thanks

**Pull request merged**